ChangeLog.md

```diff
@@ -1,5 +1,12 @@
 # Change Log - oav
 
+## 08/17/2021 2.8.0
+
+- Isolate RP when running transform on cache model in live validator so that the transform won't fail on global cache level
```

Comment:
Which file is for isolating RP when running transform?

Reply:
The logic was added to each global transformer to continue other object transforms when hit error in transforming current object.